lib/type.ts

```diff
@@ -16,3 +16,20 @@ export interface PostProps {
   article: Articles;
   [key: string]: any;
 }
+
+export interface Comments {
+  nextCursor: number;
+  list: [
+    {
+      writer: {
+        image: string;
+        nickname: string;
+        id: number;
+      };
+      updatedAt: string;
+      createdAt: string;
+      content: string;
+      id: number;
+    }
+  ];
```

Comment:
다만, 가독성을 위해 Comment라는 타입을 만들고 가져와 사용하는 구조이면 더 좋을 것 같습니다.
```suggestion
  list: Comment[];
```